webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -252,6 +252,14 @@
 
                             {# Crash circumstances #}
 
+                            {% if raw.StartupCrash %}
+                            <tr title="{{ fields_desc['raw_crash.StartupCrash'] }}">
+                                <th scope="row">Startup Crash</th>
+                                <td>
+                                    <pre>True</pre>
```

Comment:
Turns out the crash I used to test the absence of the row was a content crash, so the 'StartupCrash' annotation was missing in that case. I'll fix the patch.